internal/webhook/external/github.go

```diff
@@ -184,16 +189,17 @@ func (g *githubWebhookReceiver) getHandler(requestBody []byte) http.HandlerFunc
 			return
 
 		case *gh.PushEvent:
+			qualifier = e.GetRef()
 			// TODO(krancour): GetHTMLURL() gives us a repo URL starting with
 			// https://. By refreshing Warehouses using a normalized representation of
 			// that URL, we will miss any Warehouses that are subscribed to the same
 			// repository using a different URL format.
 			repoURL = git.NormalizeURL(e.GetRepo().GetCloneURL())
 		}
 
-		logger = logger.WithValues("repoURL", repoURL)
+		logger = logger.WithValues("repoURL", repoURL, "qualifier", qualifier)
```

Comment:
I think I'd find the logs a little bit more helpful if we could be more specific than "qualifier" here. Can we use "ref" for Git pushes and "tag" for package pushes? The easiest way to do so would probably be to add one or the other directly in their respective cases in the switch above here.